### PR TITLE
Linkedin search

### PR DESCRIPTION
**Pull request by pedromoferreira**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        <developerConnection>scm:git:git@github.com:dashdash/integration-linkedin-search.git</developerConnection>
+        <tag>HEAD</tag>
+    </scm>
+
+
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot</artifactId>
+        </dependency>
```
Comment (luisrosilva) [inline pom.xml:28]: Is spring-boot needed as a dependency or only spring-context?

Comment (luisrosilva) [reply pom.xml:28]: Also, isn't camel-core missing?

**Review by luisrosilva — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (pedromoferreira) [reply pom.xml:28]: They are all coming from the integrator-pom now

Comment (pedromoferreira) [reply pom.xml:28]: spring-boot is needed for the test classes

**Pull request merged**